app/src/main/java/com/vero/tasky/auth/domain/use_case/Authenticate.kt

```diff
@@ -0,0 +1,12 @@
+package com.vero.tasky.auth.domain.use_case
```

Comment:
Kotlin linter isn't a big fan of having a "_" in a package name
Maybe you can rename it to "usecase" or "usecases"